.github/workflows/markdown-link-check-config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "ignorePatterns": [
+        { "pattern": "^https://github.com/\\S+/\\S+/(issues|pull)/[0-9]+" },
+        { "pattern": "^mailto:" },
+        { "pattern": "^https://circleci.com/workflow-run/02d8b5fb-bc7f-404c-9051-68307c124649" }
```

Comment:
not sure where this one is coming from ?

Reply:
https://github.com/weaveworks/eksctl/blame/master/CONTRIBUTING.md#L318